axiom/nr_php_packages.h

```diff
@@ -149,8 +159,9 @@ static inline int nr_php_packages_has_package(nr_php_packages_t* h,
  *
  * Returns : Returns pointer to php package if the package exists or NULL
  */
-static inline nr_php_package_t* nr_php_packages_get_package(nr_php_packages_t* php_packages,
-                                              const char* package_name) {
+static inline nr_php_package_t* nr_php_packages_get_package(
+    nr_php_packages_t* php_packages,
+    const char* package_name) {
```

Comment:
Formatting only, right?

Reply:
Yes this change is solely from `git-clang-format`.